supertokens_python/recipe/session/utils.py

```diff
@@ -418,4 +455,5 @@ def validate_and_normalise_user_input(
         app_info.framework,
         app_info.mode,
         jwt,
+        invalid_claim_status_code if (invalid_claim_status_code is not None) else 403,
```

Comment:
This seems to be missing from the node repo as well, but we should be checking if this equals sessionExpiredStatusCode and throw if it does.

Reply:
Okay. TODO.

Reply:
TODO